### PR TITLE
Version 0.12.1

### PR DESCRIPTION
**Pull request by fingolfin**

### CODE REVIEW

**Comment by fingolfin** [comment]
@ThomasBreuer @lgoettgens I'd like to make one more GAP.jl release on the 0.12.x track with the changes that we accumulated before working on the upgrade to 0.13.0 with GAP 4.14.0 in it (see also PR #1069)

**Comment by lgoettgens** [comment]
I am not quite sure currently if #1067 breaks the new function introduced in #1066. But this is of course no blocker for anything, but I wouldn't include in any future 0.12.2 release. (after the 0.13.0 bump everything is fine again)

**Comment by fingolfin** [comment]
@lgoettgens thanks. Just for clarification: are you saying including #1066 in 0.12.1 is fine, but #1067 should be delayed until 0.13.0?

**Comment by lgoettgens** [comment]
Yes exactly. #1067 needs to be either in the same release as #1066 (aka 0.12.1) or a breaking release apart (aka 0.13.0). It should not be put in a later patch release (aka 0.12.2).

I can probably say a bit more once I am back at my computer and rebased #1067.

**Comment by lgoettgens** [comment]
Now that #1067 is rebased, we can see that it (in its current form) does not touch `GAP.Packages.build` at all. So you can ignore everything that I wrote here about #1066 and #1067.

**Comment by fingolfin** [comment]
@ThomasBreuer any objections to this PR?

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changes in GAP.jl
 
+## Version 0.12.1 (released 2024-12-07)
+
+- Add `GAP.Packages.build(name)`
+- Add iteration over GAP iterators
+- Reintroduce `julia_to_gap` for backward compatibility
+- Improve evalstr to show syntax warnings
```
Comment (fingolfin) [inline CHANGES.md:8]:
```suggestion
- Improve `evalstr` to show syntax warnings
```

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Changes in GAP.jl
 
+## Version 0.12.1 (released 2024-12-07)
```
Comment (lgoettgens) [inline CHANGES.md:3]:
```suggestion
## Version 0.12.1 (released 2024-12-09)
```

File: pkg/JuliaExperimental/PackageInfo.g [hunk 3]
```diff
@@ -10,8 +10,8 @@ SetPackageInfo( rec(
 
 PackageName := "JuliaExperimental",
 Subtitle := "Experimental code for the GAP Julia integration",
-Version := "0.12.0",
-Date := "25/09/2024", # dd/mm/yyyy format
+Version := "0.12.1",
+Date := "07/12/2024", # dd/mm/yyyy format
```
Comment (lgoettgens) [inline pkg/JuliaExperimental/PackageInfo.g:14]:
```suggestion
Date := "09/12/2024", # dd/mm/yyyy format
```

File: pkg/JuliaInterface/PackageInfo.g [hunk 4]
```diff
@@ -10,8 +10,8 @@ SetPackageInfo( rec(
 
 PackageName := "JuliaInterface",
 Subtitle := "Interface to Julia",
-Version := "0.12.0",
-Date := "25/09/2024", # dd/mm/yyyy format
+Version := "0.12.1",
+Date := "07/12/2024", # dd/mm/yyyy format
```
Comment (lgoettgens) [inline pkg/JuliaInterface/PackageInfo.g:14]:
```suggestion
Date := "09/12/2024", # dd/mm/yyyy format
```

**Review by ThomasBreuer — APPROVED** [review]
Looks good, thanks.

**Pull request merged**